packages/compiler/core/binder.ts

```diff
@@ -140,6 +140,8 @@ export function createBinder(program: Program, options: BinderOptions = {}): Bin
           nsParts.push(...memberNs.split("."));
         }
 
+        const namespace = nsParts.join(".");
```

Comment:
I can clean this up after the refactor merges, which I believe will add ability to get proper namespace symbol. I'll wait for that PR to merge before merging this one to avoid unnecessary conflict.

Reply:
Storing the namespace name on the symbol shouldn't be necessary post-refactor - once you have a symbol, you can walk its parents to get the FQI.

Reply:
Yep, that's my plan :)